cl.conf

```diff
@@ -52,6 +52,16 @@ systemd:
 
         [Install]
         WantedBy=multi-user.target
+networkd:
+  units:
+    - name: 100-eth1.network
+      contents: |
+        [Match]
+        Name=eth1
+
+        [Network]
+        Domains=vagrant.amobee.com
+        Address=192.168.42.50/24
```

Comment:
Do we have to include the 'Address' field here? I would expect that not to be necessary.

Reply:
actually we don't, it's already set.